test/support/html_test_helpers.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Phoenix.LiveView.HtmlTestHelpers do
+  @moduledoc false
+
+  require EasyHTML
+
+  defmacro sigil_X({:<<>>, _, [binary]}, []) do
+    Macro.escape(EasyHTML.parse!(binary))
+  end
+
+  def t2h(template) do
```

Comment:
Template to HTML, but if you prefer a different name

Reply:
maybe `to_html` ?

Reply:
We already have to_html in the `DOM` module so `t2h` is fine here. We could eventually align the entire DOM module to use EasyHTML structures, but separate interface via t2h is fine for now.